cfme/tests/control/test_actions.py

```diff
@@ -250,21 +190,15 @@ def vm_on(vm):
     """ Ensures that the VM is on when the control goes to the test."""
     vm.mgmt.wait_for_steady_state()
     vm.mgmt.ensure_state(VmState.RUNNING)
-    # Make sure the state is consistent
-    vm.crud.refresh_relationships(from_details=True)
-    vm.crud.wait_for_vm_state_change(desired_state=vm.crud.STATE_ON, from_details=True)
-    return vm
+    return
 
 
 @pytest.fixture(scope="function")
-def vm_off(provider, vm):
+def vm_off(vm):
     """ Ensures that the VM is off when the control goes to the test."""
     vm.mgmt.wait_for_steady_state()
     vm.mgmt.ensure_state(VmState.STOPPED)
-    # Make sure the state is consistent
-    vm.crud.refresh_relationships(from_details=True)
-    vm.crud.wait_for_vm_state_change(desired_state=vm.crud.STATE_OFF, from_details=True)
-    return vm
+    return
```

Comment:
Question:
- Same here for ```return``` 
- Don't we need to ```wait_for_vm_state_change```(here fixture doc string says that VM state should be OFF)?

Reply:
I was seeing failures related to `wait_for_vm_state_change` when running these tests. However, they may have been remedied by making `vm` function-scoped. However, since the tests are passing, I see no need to add this extra check back in there.